inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -1166,18 +1166,10 @@ MKLDNNNode* MKLDNNNode::NodesFactory::create(const std::shared_ptr<ngraph::Node>
                                              const MKLDNNExtensionManager::Ptr& extMgr, MKLDNNWeightsSharing::Ptr &w_cache) {
     MKLDNNNode *newNode = nullptr;
     std::string errorMessage;
-    try {
+    {
```

Comment:
Yes, but it seems we still catch (do we actually throw in this case?) exception in the case 2.
I mean in general we should throw only in the 3rd try-catch block as you mentioned.
But we catch NotImplemented in case 2 (so that means we throw NotImplemented from some of the nodes).
And the question is can we avoid dealing with exception in this case 2.

Reply:
Ok, I agree, but it is decided that it should be performed as a separate  activity.